Makefile

```diff
@@ -151,7 +151,7 @@ build-conbench-container-image: set-build-info
 deploy-on-minikube:
 	minikube status || minikube status --profile mk-conbench
 	mkdir -p _build
-	cp ci/minikube/deploy-conbench.template.yml _build/deploy-conbench.yml
+	/bin/cp ci/minikube/deploy-conbench.template.yml _build/deploy-conbench.yml
```

Comment:
On many systems `cp` is aliased to `cp -i`, i.e. it does not overwrite -- we want the overwrite to happen, though.